src/main/java/com/google/sps/data/SampleData.java

```diff
@@ -93,6 +93,11 @@
                 new ArrayList<String> (Arrays.asList("0")), new ArrayList<TutorSession> (Arrays.asList()), "4")
```

Comment:
Now that we can search for users, we should add a link on tutors' profiles to their availability page so students can schedule session directly from there instead of searching from the homepage.

Reply:
good idea! I'll do this a new PR after this is merged then